test/metabase/dashboard_subscription_test.clj

```diff
@@ -175,21 +176,28 @@
                                          :description   "Linked Dashboard desc"
                                          :collection_id coll-id}
        DashboardCard _                  {:dashboard_id           dashboard-id
+                                         :row                    0
```

Comment:
Fix flaky for cases where the cards order in rendered and expected are different.

Fail example: https://github.com/metabase/metabase/actions/runs/4317008327/jobs/7533714886#step:4:576

This is because we use a [comparator](https://github.com/metabase/metabase/blob/0aaba1eea1216ceb0168274c5e459ad79009471e/src/metabase/pulse.clj#L85) on `row` or `col` but these dashcards all has `row==col==0`, so I guess depends on the env of the test VM, it'll produce different results